### ISSUE TITLE
Pull compensation for fills

### ISSUE THREAD

**Issue by lexelby** [inkstitch/inkstitch#179]
This one came from the discussion of fill outsets (#178).  Add a pull compensation feature for auto-fill.  This would expand each row of stitches by the specified amount.

**Comment by fnavas00** [comment]
@lexelby im facing issues with most recent designs. Ink/stitch will freeze whenever I try to apply params for certain paths I just received this error message

"path459: error: Error during autofill!  This means that there is a problem with Ink/Stitch.

If you'd like to help us make Ink/Stitch better, please paste this whole message into a new issue at: https://github.com/inkstitch/inkstitch/issues/new

Traceback (most recent call last):
  File "lib/elements/auto_fill.py", line 218, in to_patches
  File "lib/debug.py", line 125, in decorated
  File "lib/stitches/auto_fill.py", line 64, in auto_fill
  File "lib/debug.py", line 125, in decorated
  File "lib/stitches/auto_fill.py", line 568, in path_to_stitches
  File "lib/stitches/auto_fill.py", line 533, in travel
  File "site-packages/networkx/algorithms/shortest_paths/generic.py", line 172, in shortest_path
  File "site-packages/networkx/algorithms/shortest_paths/weighted.py", line 163, in dijkstra_path
  File "site-packages/networkx/algorithms/shortest_paths/weighted.py", line 472, in single_source_dijkstra
  File "site-packages/networkx/algorithms/shortest_paths/weighted.py", line 738, in multi_source_dijkstra
NetworkXNoPath: No path to (328.0663330529999, 253.2355051856989)."

**Comment by lexelby** [comment]
Yikes!  That's no good.  If you run Troubleshoot Objects, does it show any problems?

As that error says, you have found a bug.  If you would be willing to post an SVG that gives that error, I'd greatly appreciate it, because it might help us fix the bug.

**Comment by fnavas00** [comment]
@lexelby I don't know what happened I restarted inkscape and tried to re-set the params and its working now.. but originally I was working on an SVG i wanted to change the params for the text objects but it kept freezing like in issue #546 so i opened a new file in inkscape, imported the previous svg that was freezing, I changed the text params in the new file (no issue), then tried to change the fill params and it kept showing the error message.

Here is the the file:

[AESTHETIC(text)12312321 copy.svg.zip](https://github.com/inkstitch/inkstitch/files/3688780/AESTHETIC.text.12312321.copy.svg.zip)

Also here is the finished result. I followed your suggestions for the fill.. made some pull comp adjustments to my satin columns its not perfect but definitely looks a-lot better compared to the SewArt version I did. Im assuming a lot of the issues are because im stitching on a t-shirt

![Comparison  Ink_Stitch vs SewArt](https://user-images.githubusercontent.com/50570593/66179049-0dbe4180-e62d-11e9-8890-680f89430594.jpg)
